data-es6.js

```diff
@@ -5491,6 +5491,7 @@ exports.tests = [
         chrome41: true,
         safari9: true,
         safari12: false,
+        safari12_1: true,
```

Comment:
Unless 12.1 is newer than TP 76, this is incorrect.
<img width="527" alt="screen shot 2019-02-28 at 3 15 18 pm" src="https://user-images.githubusercontent.com/112982/53595609-b4a89f80-3b6b-11e9-9c1b-21bd394d2863.png">

Reply:
It's Version 12.1 (14607.1.35.1)
Also my Safari TP is Release 76 (Safari 12.2, WebKit 14608.1.6.2)

Reply:
<img width="704" alt="image" src="https://user-images.githubusercontent.com/1507086/53596017-d4fa4d80-3baf-11e9-81ad-4b5974c5cde7.png">

Reply:
It's a memory bug that's impossible to write a test for. The general semantics are all that is tested.

Actual bug: https://bugs.webkit.org/show_bug.cgi?id=190756
Actual reproduction: https://jsbin.com/peniburaho/1/edit?html,console (you'll have to wait, possibly use other tabs)

Reply:
Can't reproduce on SF 12.1
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/1507086/53596479-e42dcb00-3bb0-11e9-9c83-ff80d6fbdcf4.png">

Reply:
I left a comment in WebKit Bugzilla

Reply:
Hmm I was able to reproduce it once
<img width="372" alt="image" src="https://user-images.githubusercontent.com/1507086/53596846-c6149a80-3bb1-11e9-845a-cc318123753b.png">
After that it works again

Reply:
That suggests it's not fixed.

Either way maybe let's just remove it from this PR, which per the title it's out of scope for, and we can discuss it in a separate PR?

Reply:
It's definitely not fixed, I can reproduce it most times in TP. If we could reproduce with 100%, I would have written the test better. 😉

Reply:
I was able to catch it second time
<img width="447" alt="image" src="https://user-images.githubusercontent.com/1507086/53597245-be092a80-3bb2-11e9-878b-bc967e76e7aa.png">